docs/topics/mta-web-downloading-an-analysis-report.adoc

```diff
@@ -11,17 +11,12 @@ For your convenience, you can download analysis reports. By default, this option
 .Procedure
 
 . In *Administration* view, click *General*.
-. Toggle the *Enable downloads for HTML reports* and *Enable downloads for CSV reports* switches.
+. Toggle the *Allow reports to be downloaded after running an analysis.*  switch.
```

Comment:
Is the period following "to be downloaded after running an analysis" needed? is it seen in the UI?

Reply:
@emarcusRH  Yes, that why i put the period here.